app/controllers/ops_controller/settings/automate_schedules.rb

```diff
@@ -17,10 +17,24 @@ def automate_schedules_set_vars
     }
   end
 
+  def name_included?(klass)
```

Comment:
Do you really need this as a separate method?

Reply:
No, I don't really need this as a separate method. The method is used just for making the code more simple to read. Should I change this?

Reply:
Yes, makes no sense to have it extracted...the method where it is being used is short enough. If you really want to have it more descriptive, add a comment.